pcdsdevices/attenuator.py

```diff
@@ -369,24 +391,22 @@ def _make_att_classes(max_filters, base, name):
             att_filters['filter{}'.format(n)] = comp
 
         cls_name = '{}{}'.format(name, i)
-        cls = type(cls_name, (base,), att_filters)
-        # Store the number of filters
+        cls = type(cls_name, (base, base_3rd_harmonic,), att_filters)
         cls.num_att = i
         att_classes[i] = cls
     return att_classes
 
 
-_att_classes = _make_att_classes(MAX_FILTERS, AttBase, 'Attenuator')
-_att3_classes = _make_att_classes(MAX_FILTERS, AttBase3rd, 'Attenuator3rd')
+_att_classes = _make_att_classes(
+    MAX_FILTERS, AttBase, AttBaseWith3rdHarmonic, 'Attenuator')
```

Comment:
I'm still not sure if this is safe - this is assuming that users will not create a Fee attenuator using this function, but I think that's the case anyway?

Reply:
The FEE attenuator is a one-off device.

This factory function is used because the LusiAtt devices can come in any number of flavors based on the number of silicon filters in the device. E.g. it's me being lazy and not wanting to hand-make a bunch of classes.

There shouldn't be any danger in doing this.